ddtrace/propagation/_database_monitoring.py

```diff
@@ -0,0 +1,82 @@
+from typing import TYPE_CHECKING
+
+from ddtrace.internal.logger import get_logger
+from ddtrace.vendor.sqlcommenter import generate_sql_comment as _generate_sql_comment
+
+from ..internal.utils import get_argument_value
+from ..internal.utils import set_argument_value
+from ..settings import _config as dd_config
+from ..settings._database_monitoring import dbm_config
+
+
+if TYPE_CHECKING:
+    from typing import Optional
+
+    from ddtrace import Span
+
+DBM_PARENT_SERVICE_NAME_KEY = "ddps"
+DBM_DATABASE_SERVICE_NAME_KEY = "dddbs"
+DBM_ENVIRONMENT_KEY = "dde"
+DBM_VERSION_KEY = "ddpv"
+DBM_TRACE_PARENT_KEY = "traceparent"
+DBM_TRACE_INJECTED_TAG = "_dd.dbm_trace_injected"
+
+
+log = get_logger(__name__)
+
+
+def _default_sql_injector(dbm_comment, sql_statement):
+    try:
```

Comment:
```suggestion
    # type: (str, str) -> str
    try:
```
Should we add typing?